### ISSUE TITLE
path and flush_deletes

### ISSUE THREAD

**Issue by lnxpck** [GoGoCarl/paperclip-cloudinary#3]
Hi !

I also have an issue with delete contents.
It seems path is not correct in flush_deletes.

I add this logs lines in the flush_deletes to see what's wrong :

```
def flush_deletes
        log("deleting...")
        @queued_for_delete.each do |path|
          log("deleting #{path}")
          ::Cloudinary::Uploader.destroy path
        end

        @queued_for_delete.clear
      end
```

And here are the logs, and file is not deleted : 

```
[paperclip] deleting...
[paperclip] deleting avatar_96788ee0-1e8b-0134-7d9a-0242ac110001.png
```

When i tried with the rails Console, it work's fine if path is without extension : 

```
➜  rails console                                                                                                                                                                                                    
Running via Spring preloader in process 906
Loading development environment (Rails 4.2.6)
irb(main):001:0> Cloudinary::Uploader.destroy('45d3b060-1e8a-0134-578f-525400c10e34.png')
=> {"result"=>"not found"}
irb(main):002:0> Cloudinary::Uploader.destroy('45d3b060-1e8a-0134-578f-525400c10e34')
=> {"result"=>"ok"}
```

Is this an issue or is it something wrong in my configuration ?

Thx.
Patrik

**Comment by GoGoCarl** [comment]
Hi @plesaint,

Thanks for this. I believe that is an issue, not your config. I'll release a fix for this in 1.2 as well.

**Comment by GoGoCarl** [comment]
@plesaint Hi Patrick, a fix for this has been merged and released under version 1.2.0. Thanks for reporting!
